test/unit/test.natural_language_understanding.v1.js

```diff
@@ -37,11 +37,23 @@ describe('natural_language_understanding', function() {
     });
     done();
   });
+  
+  it('2016_01_23 version should work', function(done) {
+    nock(watson.NaturalLanguageUnderstandingV1.URL)
+      .post('/v1/analyze?version=' + watson.NaturalLanguageUnderstandingV1.VERSION_DATE_2016_01_23)
+      .reply(200, {});
+
+    const options = {
+      features: { concepts: {}, keywords: {} },
+      text: 'hello, this is a test'
+    };
+
+    nlu.analyze(options, done);
```

Comment:
This nlu instance is the one with the new version date, so it won't hit the nock you set up above. You'll need to create a new instance just for this test.

If this test already passes as-is then I think we must be doing something else wrong...